### ISSUE TITLE
Get mobile network type for ConnectivityManager.TYPE_MOBILE

### ISSUE THREAD

**Issue by suleymanccelik** [pwittchen/NetworkEvents#63]
Some network operations are limited for mobile edge network. Can you add mobile network types to connection status? You can get mobile network types with TelephonyManager class. 

```java
TelephonyManager tm =
                (TelephonyManager) context.getSystemService(Context.TELEPHONY_SERVICE);
if (activeNetwork.getType() == ConnectivityManager.TYPE_MOBILE) {
                switch (tm.getNetworkType()) {
                    case (TelephonyManager.NETWORK_TYPE_LTE |
                            TelephonyManager.NETWORK_TYPE_HSPAP):
                        Log.i("mobile-network-type "," 3g mobile type");
                        return TYPE_MOBILE_HSPAP;
                    case (TelephonyManager.NETWORK_TYPE_EDGE |
                            TelephonyManager.NETWORK_TYPE_GPRS):
                        Log.i("mobile-network-type "," edge mobile type");
                        return TYPE_MOBILE_EDGE;
                    default:
                        Log.i("mobile-network-type","unknown mobile type");
                        return TYPE_UNKNOWN;
                }
            }
```

**Comment by pwittchen** [comment]
Thanks for reporting that issue. Detecting type of the mobile network is not part of library functionality. I'll consider adding such feature.

**Comment by pwittchen** [comment]
Done. It will be available in the next release.

**Comment by suleymanccelik** [comment]
Thanks @pwittchen

**Comment by pwittchen** [comment]
It's available now in version 2.0.0 of the library.
When you [subscribe for the `ConnectivityChanged` event](https://github.com/pwittchen/NetworkEvents#subscribe-for-the-events) you can get mobile network type from the `ConnectivityChanged` event object as follows:
```java
@Subscribe
public void onEvent(ConnectivityChanged event) {
  event.getMobileNetworkType()
}
```
Please note, that mobile network type will be known, only if `ConnectivityStatus` is equal to `MOBILE_CONNECTED`. In other cases it will be unknown.
